evaluators/jsonpath-evaluator/features/org.wso2.carbon.gateway.evaluators.jsonpathevaluator.feature/pom.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ /*
+  ~ * Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~ *
+  ~ * Licensed under the Apache License, Version 2.0 (the "License");
+  ~ * you may not use this file except in compliance with the License.
+  ~ * You may obtain a copy of the License at
+  ~ *
+  ~ * http://www.apache.org/licenses/LICENSE-2.0
+  ~ *
+  ~ * Unless required by applicable law or agreed to in writing, software
+  ~ * distributed under the License is distributed on an "AS IS" BASIS,
+  ~ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ * See the License for the specific language governing permissions and
+  ~ * limitations under the License.
+  ~ */
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>org.wso2.carbon.gateway.evaluators.jsonpathevaluator.parent</artifactId>
+        <groupId>org.wso2.carbon.gateway</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.wso2.carbon.gateway.evaluators</groupId>
+    <artifactId>org.wso2.carbon.gateway.evaluators.jsonpathevaluator.feature</artifactId>
+    <packaging>carbon-feature</packaging>
+    <name>WSO2 Carbon - Gateway JSONPath Evaluator Feature</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.carbon.gateway.evaluators</groupId>
+            <artifactId>org.wso2.carbon.gateway.evaluators.jsonpathevaluator</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```

Comment:
Can't we use "carbon.gw.version" property here? Hard coding versions is not recommended in child poms since it makes troublesome when doing a release.